### ISSUE TITLE
Search query WebObjects/MZStore.woa/wa/search returns http 503 error 

### ISSUE THREAD

**Issue by KpoxaDroid** [facundoolano/app-store-scraper#129]
This search query
https://itunes.apple.com/WebObjects/MZStore.woa/wa/search?clientApplication=Software&media=software

returns 503 error with all headers set correctly.

* Operating System:
* Node version:
* app-play-scraper version:

### Description: 
Describe your issue here

### Example code:
```js
// Put code to reproduce the issue here
```

### Error message:

```
Put error message here
```

**Comment by cancuta** [comment]
after removing "&media=software" particle the response code became 200 OK

**Comment by KpoxaDroid** [comment]
no, it doesn't help.

**Comment by Steffen1986** [comment]
I have this issue as well. By the way is there any documentation for using this endpoint? 
Could be that it is a request limit issue.

**Comment by KpoxaDroid** [comment]
No, i tried this call with http proxies and even with different socks5, its just don't work anymore.

**Comment by Steffen1986** [comment]
I think it's not due to maintenance. Cause removing "&media=software" worked yesterday but not today. Are you sure that there is no request limit?

**Comment by Djip** [comment]
Go into `search.js` in the libs folder, and change line 5 to

`const BASE_URL = 'https://search.itunes.apple.com/WebObjects/MZSearch.woa/wa/search?clientApplication=Software&media=software&term=';`

**Comment by Steffen1986** [comment]
Thank you @Djip . Any ideas what's going on with previous url? Usually there is more information when receiving a 503 (Retry-After in header is missing).

**Comment by marcoadasilvaa** [comment]
Same error here. I am looking official reference of apple:

https://affiliate.itunes.apple.com/resources/documentation/itunes-store-web-service-search-api/

**Comment by KpoxaDroid** [comment]
> Go into `search.js` in the libs folder, and change line 5 to
> 
> `const BASE_URL = 'https://search.itunes.apple.com/WebObjects/MZSearch.woa/wa/search?clientApplication=Software&media=software&term=';`

this one is working perfectly!

**Comment by Djip** [comment]
> 
> 
> Thank you @Djip . Any ideas what's going on with previous url? Usually there is more information when receiving a 503 (Retry-After in header is missing).

They probably hard swapped it out. Since the only difference is that they added `search.` in front of the url

**Comment by Steffen1986** [comment]
> > Thank you @Djip . Any ideas what's going on with previous url? Usually there is more information when receiving a 503 (Retry-After in header is missing).
> 
> They probably hard swapped it out. Since the only difference is that they added `search.` in front of the url

Strange. Searching for iPadSoftware works with the old url as well: https://itunes.apple.com/WebObjects/MZStore.woa/wa/search?clientApplication=iPadSoftware&term= . Hopefully they will not close other endpoints due to inexplicable reasons.

**Comment by benfontan** [comment]
Hi, I'm having the same issue and changing the URL fixed it. However I'm installing it in a cloud function which automatically gets package from npm. Is this included in a released yet?
Thanks @gamebenchjake !

**Comment by Djip** [comment]
Take a look at my fork, it has the fix pushed onto it 👍

**Comment by facundoolano** [comment]
published `app-store-scraper@0.16.3` with the fix from #134 
Please, let me know if the issue is fixed now.
